private/syntax-path.rkt

```diff
@@ -1206,6 +1207,123 @@
 
 
 
+(define (in-syntax-paths stx #:base-path [base-path empty-syntax-path])
+  (make-do-sequence
```

Comment:
Use generators instead of `make-do-sequence`. Specifically, use the `in-generator` form from `racket/generator`.